NEWS

```diff
@@ -11,6 +11,171 @@
+* Fixed probe for multi-fragment eager
+* Fixed alignment for AM rdesc init
+* Fixed perf estimation for proto v2
+* Fixed CM wireup with proto v2
+* Fixed EP discard flow during fast-forward
+* Fixed datatype issue in TAG send
+* Fixed EP refcount overflow
+* Fixed EP error handling flow
+* Fixed wire compatibility in address unpacking
+* Fixed UCP EP destroy which owns send request when invalidation is in-progress
+* Fixed fragmented proto v2
+* Fixed UCP address v2 packing/unpacking and usage of seg_size
+* Fixed possible purge of requests after discarding is in-progress after error handling
+* Fixed request purging during UCT EPs destroy
+* Fixed error handling of connecting p2p lanes during WIREUP phase
+* Fixed UCP endpoint use after free
+#### UCT
+* Fixed ABI break of uct_ep_params_t
+* Fixed common intra-node keepalive protocol
+* Fixed a typo UCT_PERF_ATTR_FIELD_REMOTE_SYS_DEIVCE -> UCT_PERF_ATTR_FIELD_REMOTE_SYS_DEVICE
+* Fixed potential crash on MD mem alloc
+* Disabled PEER_FAILURE capability for XPMEM
+#### RDMA CORE (IB, ROCE, etc.)
+* Fixed 2G aligned MR registration
+* Fixed FC_HARD_REQ resending
+* Fixed remote access to invalidated MR
+* Fixed max_rd_atomic_dc value for DV
+* Fixed DC handshake logic
+* Fixed error handling flows
+* Fixed flush(CANCEL) with UD and DC transports
+* Fixed multi-path handling for passive endpoint with UD transport
+* Fixed attributes for DV QP creation
+* Fixed device query
+* Fixed memory leak in case of disabling RDMA transport
+* Fixed dci->pool_index initialization
+* Fixed fallback if port speed not detected
+* Fixed tag offload recv for inlined data
+* Fixed PKEY index initialization
+* Disabled mlx5 ifaces on verbs MD
+#### TCP
+* Fixed flush(CANCEL)
+* Fixed close protocol when UCT EP pairs have only RX capability
+* Fixed query local/remote saddr
+#### GPU (CUDA, ROCM)
+* Fixed a bug in invalidating address range in CUDA_IPC
+* Fixed CUDA context caching and cleanup
+* Fixed ROCM initialization
+* Fixed ROCM components compilation
+* Fixed IPC tls reachability check
+* Fixed ROCM memory type detection
+* Use ROCM remote_agent if available
+#### KNEM
+* Fixed memory registration cost
+#### UCM
+* Fixed potential hang on init
+#### UCS
+* Fixed name shadow problem in CentOS6.x
+#### Tools
+* Print stream API limits and handle stream feature in ucx_info
+* Replaced ucp_ep_close_nb by ucp_ep_close_nbx in examples
+* Replaced completed field by checking UCS status in io_demo
+#### JAVA
+* Throw exception if ucp_mem_query failed
+#### GO
+* Disabled go bindings in rpmbuild
+* Fixed configure if can't find go compiler
+* Standalone performance benchmark
+* Increased port range + make it dependent from agent_id
+* Checked compiler minimum version
+* Set GOCACHE a local directory that is cleared for each job in CI
+* Disabled module for goperftest
+* Fixed OOS build
+* Wait for thread to write to the channel
```

Comment:
Needs some context

Reply:
this is minor test change, so removing it